### DISCUSSION TITLE
Please clarify the plans for paid subscriptions

### DISCUSSION THREAD

**Discussion by SoMuchForSubtlety** [getseabird/seabird#28] [General]
From the README:
> We plan to distribute this application with an semi-optional yearly subscription price to support development.

Do you plan on holding back features from this repository like lens does?

**Comment by jgillich** [comment]
Not at all, this was born out of frustration with Lens. It's fully open source with no limitations. But I'd like to try to make this sustainable by asking users to buy a license via a small banner on the start page. See 45a316acf99f76ea236c355f61f847f50050af73 for a mockup

By paying, you'd make the banner go away and you'd get direct priority support, that's it. There won't be any online services or enterprise features or anything like that, but I wouldn't mind paying 0.5-2 people to work on this project long-term.

**Reply by SoMuchForSubtlety** [reply]
Makes sense, thanks.

**Comment by ogarcia** [comment]
Instead of putting __"Your free trial expires in ∞ days"__ I would put __""Support the development of this software"__ and in the button I would put __"Donate"__.

Once inside the "Donate to Seabird" window I would explain that this is free software but since you have to eat that if they want to remove the donate banner that they donate something. You put the minimum donation amount you are interested in and if they donate more, welcome (this allows you to be paid more than you stipulate for the license).

The inner workings may be exactly the same as they are now, but a voluntary donation in exchange for removing the banner is better than talking about licensing, which can lead to confusion about open source.

Obviously, this is just a suggestion.

**Reply by jgillich** [reply]
Thanks for the feedback! The terminology is by design, I wanted to be clear that it's not asking for charity, but for payment in an equal exchange. You'd also get extended support with a subscription, which usually isn't the case with a donation. I'll see about replacing mentions of license with subscription so it doesn't cause confusion with regards to the code license.

**Reply by ogarcia** [reply]
I understand, so maybe the text should be similar to "Subscribe and get preferential support and other benefits" and the button should read "Subscribe".
